http-clients/src/test/java/org/triplea/http/client/web/socket/WebSocketConnectionTest.java

```diff
@@ -154,52 +199,36 @@ private void verifyErrorHandlerWasCalled() {
       verify(errorHandler).accept(any());
     }
 
-    @Test
-    @DisplayName(
-        "Verify connect failing with exception invokes error handler and pinger is not running")
```

Comment:
Why is this test no longer applicable?

Reply:
After my changes there was no longer an InterruptedException to be thrown so the "interrupted" and the "errored" test cases wer suddenly the same